README.md

```diff
@@ -100,6 +139,37 @@ public class Console {
 }
 ```
 
+### Configure a Proxy
+
+The Confidential Client accepts an additional optional parameter called `RequestOptions`. This can be created to specify a proxy for the client to use. Below is an example of how to do this:
+
+```java
+Proxy proxy = new Proxy(Proxy.Type.HTTP, new InetSocketAddress("127.0.0.1", 8080));
+RequestOptions requestOptions = RequestOptions.builder().proxy(proxy).build();
+
+// Pass this into client
+ConfidentialClient confidentialClient = new ConfidentialClient("./path/to/config.json", requestOptions);
+```
+
+### Custom SSL Certificate
+
+If you have proxies or firewalls which are using custom TLS certificates, you are able to [modify the Java Runtime Environment keystore](https://docs.plm.automation.siemens.com/content/polarion/20/help/en_US/polarion_windows_installation/manually_updating_third_party_software/import_a_certificate_to_the_java_keystore.html) so that the request library is able to verify the validity of that certificate.
+
+It is also possible to have a custom SSL configuration with the code example given below.
+
+```java
+SSLContext sslContext = SSLContext.getInstance("SSL");
+sslContext.init(...);
```

Comment:
maybe add a comment that this needs to be configured to the application needs.

same for the hostnameverifier part